lib/desired-caps.js

```diff
@@ -0,0 +1,87 @@
+const desiredCapConstraints = {
+  platformName: {
+    presence: true,
+    isString: true,
+    inclusion: ['iOS']
```

Comment:
Good point. We will only instantiate the iOS driver if the `platformName` is `iOS`, so there is no real need for this check. I'm not sure if it would be best to have it anyway, as documentation of the specific caps (since technically it _is_ required).